02_bubble/sirigithub/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func bubbleSort(s []int) []int {
+	n := len(s)
+	arrayToSort := make([]int, n)
+	copy(arrayToSort, s)
+
+	for i := 0; i < n; i++ {
```

Comment:
There is no point counting from `0` here, as the inner loop does nothing when `i == 0`, since it starts with `j := i` and only loops when `j > 0`

Reply:
@CameronSchafer thats a good point
Done. Updated code to iterate from i=1.